models/stg_hubspot__merged_deal.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_sales_enabled','hubspot_deal_enabled'])) }}
```

Comment:
Should we also consider adding a variable to have this disabled via a new variable if the `merged_deal` table is not present in the source?

Do we know how common this table is?